portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,164 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+*{
+    box-sizing: border-box;
 }
 
-#greeting-container {
-  margin-top: 20px;
+nav {
+    background-color: black;
```

Comment:
Alphabetize the css rules in this file: https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order

Reply:
Oh ok that makes sense sorry I forgot to take a look at the style guides for Html and Css before starting